sky/provision/kubernetes/utils.py

```diff
@@ -187,7 +187,7 @@ def get_accelerator_from_label_value(cls, value: str) -> str:
             return value.replace('nvidia-tesla-', '').upper()
         elif value.startswith('nvidia-'):
             acc = value.replace('nvidia-', '').upper()
-            if acc == 'H100-80GB':
+            if acc in ['H100-80GB', 'H100-MEGA-80GB']:
                 # H100 is named as H100-80GB in GKE.
```

Comment:
Could we explain the difference between H100 and H100-MEGA in the comment? It would also be great to add a TODO here for possible future distinction of the different bandwidths.
```
H100 is named H100-80GB or H100-MEGA-80GB in GKE, where
the latter has improved bandwidth. See a3-mega instances on GCP.
TODO: we do not distinguish the two GPUs for simplicity, but we can
evaluate whether we should distinguish them based on users' requests.
```